packages/client/src/offline/reducer.ts

```diff
@@ -274,21 +275,29 @@ function reducer(
       }
       return loop(state, dataLoadingCmds)
     }
-
+    case actions.UPDATE_OFFLINE_CONFIG: {
+      return loop(
+        { ...state },
+        Cmd.list([
+          CONFIG_CMD,
```

Comment:
I don't think we need to call the api for the `ApplicationConfig` here as we can already have that updated data as the payload for this action. I have left a comment from where the `UPDATE_OFFLINE_CONFIG` is being called so maybe that will make things a bit more clear.

Reply:
Yeah. Your thought is logical. I have changed the way it changes the value on redux and index.